addons/ai/XEH_postInit.sqf

```diff
@@ -68,3 +68,11 @@
     params ["_unit", "_mode"];
     _unit enableGunLights _mode;
 }] call CBA_fnc_addEventHandler;
+
+
+if (isServer) then {
+    ["CAManBase", "init", {
```

Comment:
Would using `"postInit"` instead of `"init"` make a difference when it comes to `hmd _unit` returning `""`?

Reply:
No, has been tested.